README.md

```diff
@@ -4,11 +4,14 @@
   <img alt="VMClarity Logo" src="./img/logos/VMClarity-logo-light-bg-horizontal@4x.png">
 </picture>
 
+[![Slack Invite](https://img.shields.io/badge/Slack-Join-blue?logo=slack)](https://outshift.com/slack)
```

Comment:
Lets change this link to:
```
https://outshift.slack.com/messages/vmclarity
```
so that it goes directly to the VMClarity channel

Reply:
updated!